### PR TITLE
Register Subdomain: scolder

### PR DESCRIPTION
**Pull request by ScolderCreations**
Added the domain: scolder

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/scolder.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "ScolderCreations",
+        "email": "ozor1xx@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "scolder"
```
Comment (TheBotlyNoob) [inline domains/scolder.json:9]: Maybe you meant to use your `github.io` url?
```suggestion
        "CNAME": "https://scoldercreations.github.io/"
```

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

File: domains/scolder.json (hunk 1 above)
Comment (WilliamDavidHarrison) [reply domains/scolder.json:9]: @TheBotlyNoob That's not a valid CNAME. It would be `scoldercreations.github.io`.

Comment (ScolderCreations) [reply domains/scolder.json:9]: Sorry I got that wrong, I’ll fix it.

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please add more content to your website.

**Pull request closed without merging**